introduction.md

```diff
@@ -0,0 +1 @@
+Learning a few things every day.  Today it is Github techniques...
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.